xtask/src/runchecks.rs

```diff
@@ -289,8 +290,18 @@ fn std_checks() {
 }
 
 fn check_typos() {
-    // Install typos-cli
-    cargo_install(["typos-cli", "--version", "1.16.5"].into());
+    // This path defines where typos-cl is installed on different
```

Comment:
`typos-cl` is a typo :)

Reply:
Ugh, you're right! eh eh
Can you open a new PR to fix the typo, please?